app/internal/daily_quotes.py

```diff
@@ -28,3 +28,41 @@ def quote_per_day(
     quote = session.query(Quote).filter(
         Quote.id % TOTAL_DAYS == day_num).order_by(func.random()).first()
     return quote
+
+
+def get_quotes(session, user_id):
+    """Retrieves the users' favorite quotes from the database."""
+    quotes = []
+    user_quotes = session.query(UserQuotes).filter_by(user_id=user_id).all()
+    for user_quote in user_quotes:
+        quote_id = user_quote.quote_id
+        quote = session.query(Quote).filter_by(id=quote_id).first()
```

Comment:
why not call it 
`quote = session.query(Quote).filter_by(id=user_quote.quote_id).first()`